MiniApp/Classes/Models/MiniAppInfo.swift

```diff
@@ -1,7 +1,13 @@
 /// Mini App information
-public struct MiniAppInfo {
-    var name: String
-    var description: String
-    var icon: URL
-    var appId: String
+public struct MiniAppInfo: Decodable {
+    public var id: String
+    public var name: String
+    public var description: String
+    public var icon: URL
+    public var version: Version
+}
+
+public struct Version: Decodable {
+    public var versionTag: String
```

Comment:
Ah, I didn't realize that the spec for this changed again. We will need to update this on Android...